corehq/apps/export/models/new.py

```diff
@@ -183,6 +183,7 @@ def create_from_question(cls, question, app_id, app_version, repeats):
     @classmethod
     def merge(cls, one, two):
         item = one
+        item.label = two.label  # always take the newest label
```

Comment:
i don't think we can technically know if `one` or if `two` is the newest. this feels like we're getting lucky based on how we're passing the items into the merge function. it's not obvious to me though how we can smartly resolve this besides doing something really inefficient like looking up the last_occurrences. that said i don't think this is really that big of a deal and mostly just a bonus feature, so i'm fine with it if it works